packages/grafana-eslint-rules/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@grafana/eslint-plugin",
+  "description": "ESLint rules for use within the Grafana repo. Not suitable (or supported) for external use.",
+  "version": "9.4.0-pre",
+  "main": "./index.cjs",
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
```

Comment:
Awaiting double check to make sure Apache 2 is the correct license to use here.

Reply:
Yes. It is the one we use for all of the other packages.